### ISSUE TITLE
[Error] - mac install not quite working

### ISSUE THREAD

**Issue by mixmix** [ssbc/scuttle-shell#20]
running branch `windowsInstall` withing patchbay (branch `scuttle-shell`)

I see patchbay launch, and hermes in the tray (as well as the words "scuttle-shell" taking up space in the tray!), but no content loads.

In the terminal I see : 

```
STARTING electron
LOADING config
> scuttle-shell: starting
started sbot server v13.0.3
ssb-friends: stream legacy api used
Error { notFound: true, status: 404 }
systrayhelper 0.0.4 (ded1f2ed4d30f6ca2c89a13db0bd3046c6d6d0f9 built 2018-10-20T17:30:44Z)
update-item warning!
SeqID too large. has:0 want:1
LOADING config
STARTING app
```

I'm gonna go read about that SedID error.

**Comment by cryptix** [comment]
Hrrrm... I really thought I won against this invariant. 

The problem is between the forked-systray and the native helper binary. You supply a menu as the first action and then later update entries by referencing their sequence number (or SeqID).

Ideally, it would also allow to grow the menu but that’s not handled by the json rpc interface right now...

I’m not sure why this is a problem on OS X only... scuttle-shell should create a menu with 3 entries but somehow it’s empty after init, it seems... 

I’m working on a UI-tested version of the trayhelper, which uses simulated clicking to make this all more solid.

**Comment by mixmix** [comment]
Yeah i guessed it was about the menu. Not sure where to start into those
though to help.
Would talking about it on a call help?
